src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -1,21 +1,67 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="ilmerge.props" />
 
   <PropertyGroup>
     <Description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</Description>
-    <TargetFrameworks>net45</TargetFrameworks>
-    <TargetFramework />
+    <TargetFramework>net45</TargetFramework>
     <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572</NoWarn>
     <PackProject>true</PackProject>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);CreateNupkgFromILMerge</TargetsForTfmSpecificContentInPackage>
     <Shipping>true</Shipping>
     <IncludeInVSIX>false</IncludeInVSIX>
     <XPLATProject>false</XPLATProject>
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
     <DefineConstants>$(DefineConstants);IS_DESKTOP</DefineConstants>
     <IsDesktop>true</IsDesktop>
+    <ILMergeDirectory>$(MSBuildThisFileDirectory)bin\ilmerge\</ILMergeDirectory>
```

Comment:
you probably want this to be `$(OutputPath)ilmerge\`

Reply:
started like that...however, when aftertargets="build" executes the ilmerge, outputpath doesn't exist. so chose to put under bin\ilmerge. will give it a go again...

Reply:
needed to move ilmerge to TargetsForTfmSpecificContentInPackage and that fixed my outputpath problem.